src/client/atoms.ts

```diff
@@ -0,0 +1,26 @@
+import { Prisma } from '@prisma/client'
+import { atom } from 'jotai'
+
+// Display Management
+export const loginState = atom<'login' | 'signup'>('login')
```

Comment:
i also suggest appending Atom at the end of each var name: `loginStateAtom` so that in your code when you import these you can be clear you are importing an atom.

```js
// state.js
export loginStatusAtom = atom...

// LoginPage.js

import {loginStatusAtom} from ...

function LoginPage(...) {
   const [loginStatus, setLoginStatus] = useAtom(loginStatusAtom)
}